frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -119,7 +119,7 @@ function FasterExplicitSizeUpdateDecorator(Story: Story) {
  * This value works together with FasterExplicitSizeUpdateDecorator which
  * make sure we finish resizing any ExplicitSize components the fastest.
  */
-const TIME_UNTIL_ALL_ELEMENTS_STOP_RESIZING = 1000;
+const TIME_UNTIL_ALL_ELEMENTS_STOP_RESIZING = 1500;
```

Comment:
This doesn't take 50% more time for these tests, I ran tests locally and it increases the test suite time from 4-10% or (5 seconds at most on my Macbook pro)